### DISCUSSION TITLE
how to config auto accepted

### DISCUSSION THREAD

**Discussion by bonnlee2024** [shellhub-io/shellhub#4091] [Q&A] [answered]
The device in a very secure environment. In order to improve the operating way for end users, I need to ignore the device acceptance by click.  

i want that if a device is connected, it will be automatically accepted. How can I set this up?

**Comment by heiytor** [comment]
You can create a job that runs every minute and accepts pending devices via the API using a namespace's API key.

**Comment by danielgatis** [comment, marked as answer]
Maybe this PR could help you: https://github.com/shellhub-io/shellhub/pull/6264
